Sources/Logging/LogHandler.swift

```diff
@@ -120,13 +120,21 @@ public protocol LogHandler {
     ///     - level: The log level the message was logged at.
     ///     - message: The message to log. To obtain a `String` representation call `message.description`.
     ///     - metadata: The metadata associated to this log message.
+    ///     - source: The source where the log message originated, for example the logging module.
     ///     - file: The file the log message was emitted from.
     ///     - function: The function the log line was emitted from.
     ///     - line: The line the log message was emitted from.
     func log(level: Logger.Level,
              message: Logger.Message,
              metadata: Logger.Metadata?,
-             file: String, function: String, line: UInt)
+             source: String,
+             file: String,
+             function: String,
+             line: UInt)
+
+    /// SwiftLog 1.0 compatibility method
+    @available(*, deprecated, renamed: "log(level:message:metadata:source:file:function:line:)")
+    func log(level: Logging.Logger.Level, message: Logging.Logger.Message, metadata: Logging.Logger.Metadata?, file: String, function: String, line: UInt)
```

Comment:
Looks good, but I noticed when I had implemented my own custom handler it did not report that I'm implementing a deprecated method, too bad I guess that those dont get reported.

Reply:
Yes :'(. Sad bug: https://bugs.swift.org/browse/SR-12927

Reply:
Thx for the link, subbed